test/io/test_data_handlers.py

```diff
@@ -96,6 +109,31 @@ def test_netcdf_writer(sample_dataset: xr.Dataset):
     tmp_dir.cleanup()
 
 
+def test_split_netcdf_writer(sample_dataset: xr.Dataset):
+    params = {"time_interval": 1, "time_unit": "m"}
+    # expected = sample_dataset.copy(deep=True)  # type: ignore
+    writer = SplitNetCDFWriter(parameters=recursive_instantiate(params))
+    tmp_dir = tempfile.TemporaryDirectory()
+    time_coord = [
+        datetime.strptime(x, "%Y-%m-%d %H:%M:%S")
+        for x in sample_dataset["timestamp"].data
+    ]
+    test_dataset = sample_dataset.assign_coords({"index": time_coord}).rename({"index": "time"})  # type: ignore
```

Comment:
Assigning coords to the sample dataset is a bit risky. I think this might be done in-place so it's not clear to me if this changes the dataset for the rest of the tests. If you are going to go with this approach then I would work with a copy of the sample dataset just in case. Another approach would be to create a separate `standardized_dataset` fixture and use that as input instead of standardizing the sample dataset in this test code. This approach has the benefit of making the dataset reusable if you want to add more tests to this method in the future.

Reply:
Gotcha. A fixture is probably the better bet then.